app/helpers/comments_helper.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module CommentsHelper
+  def comment_formatter(comment)
```

Comment:
I can see how other "text like places" may need a "mentions_formatter" to format their content, but it seems that this helper has now turned into a _very_ thin shell? Do we still need it?

Reply:
It's an abstraction that lives in another place which I think is the biggest pitfall.
I quite like the separation of the wrapper though, because this, and the mention formatter perform different jobs. Not even because we *might* re-use it, but because it doesn't muddle the concept of what's happening. It would be simple to do it all at once, but it becomes less clear if we begin santizing html at the same time as we're enhancing mentions.

Also- with the sanitization that happens there's a required order of operations here.

I'm happy to combine them though if it's preferred.

Reply:
it seems this is the job the Html::Pipeline was designed to handle? We can have a Comments pipeline that does whatever is needed, and build another pipeline for other types of formatters?

Reply:
Oh I think you might like where this comment lead me.